docs/security/trust/content_trust.md

```diff
@@ -221,16 +221,16 @@ Once you have a root key on your system, subsequent images repositories
 you create can use that same root key:
 
 ```bash
-$ docker push docker.io/docker/seaside:latest
-The push refers to a repository [docker.io/docker/seaside] (len: 1)
+$ docker push docker.io/<username>/<someimage>:latest
```

Comment:
Can we specify here that `<someimage>` is any image name other than `trusttest`?  Maybe we can use the image name `nottrusttest` instead?  Since this section describes the case where you are pushing a different image repository.